daprdocs/content/en/reference/components-reference/supported-secret-stores/hashicorp-vault.md

```diff
@@ -57,6 +59,7 @@ The above example uses secrets as plain strings. It is recommended to use a loca
 | vaultTokenMountPath | Y | Path to file containing token | `"path/to/file"` |
 | vaultToken | Y | [Token](https://learn.hashicorp.com/tutorials/vault/tokens) for authentication within Vault.  | `"tokenValue"` |
 | vaultKVPrefix | N | The prefix in vault. Defautls to `"dapr"` | `"dapr"`, `"myprefix"` |
+| vaultKVUsePrefix | N | If false, vaultKVPrefix is forced to be empty | `"true"`, `"false"` |
```

Comment:
Please provide more information on what this field is and does

Reply:
@AaronCrawfis thanks for the review, I hope I added a commit that clarify this argument and I should have added this commit to this pull request, let me know if I did everything correctly

Reply:
Looks great, thanks!